bigcommerce/big-design
# chore: downgrade inquirer and inquirer-autocomplete-prompt to cjs

## What?

Downgrading `inquirer` and `inquirer-autocomplete-prompt`

## Why?

version 9+ of `inquirer` and 3+ of `inquirer-autocomplete-prompt` use ESM and break the download script in production. 

## Screenshots/Screen Recordings

<img width="578" alt="Screenshot 2023-07-03 at 4 22 21 PM" src="https://github.com/bigcommerce/big-design/assets/106188209/0a9a3ad5-1133-438b-b424-45daa65efd40">

## Testing/Proof

- pull PR down 
- run `cd packages/big-design-icons && yarn && yarn run download`
- (make sure your on the right node version)
- observe download runs
- see screenshot above
